codecov/settings_prod.py

```diff
@@ -53,7 +53,9 @@
 # Redirect after authentication, update this setting with care
 CORS_ALLOWED_ORIGIN_REGEXES = []
 
-DATA_UPLOAD_MAX_MEMORY_SIZE = 15000000
+# 25MB in bytes
+DATA_UPLOAD_MAX_MEMORY_SIZE = 26214400
```

Comment:
Newbie question - are our env vars hardcoded in these files, or is there a separate config that gets fed in during the build?

Reply:
it depends! lemme ping you

Reply:
Thank you!
For anyone reading this in future, the dynamic values called via `get_config()` are as defined as here - https://github.com/codecov/k8s-v2/blob/d438b1082f3c5b1d2ed67a8520460a9635ea4dbd/charts/codecovApi/conf/env.yaml#L1

All others are hard coded